src/sentry/conf/server.py

```diff
@@ -924,6 +924,8 @@ def create_partitioned_queues(name):
     "organizations:inbox": False,
     # Return unhandled information on the issue level
     "organizations:unhandled-issue-flag": False,
+    # Adds additional filters and a new section to issue alert rules.
+    "projects:alert-filters": True,
```

Comment:
Should we also co-ordinate this with removing the feature handler from getsentry? Actually, will anybody actually have this as `False` going forward? If not, it might be worth doing a follow-up pr to remove this feature.

Reply:
oh shoot my bad let me fix the tests, and yes, moving forward no one should have this as false. I'll do a follow-up PR to clean up the code

Reply:
This will also turn on this feature in the next single-tenant deploys and selfhosted. Might be worthwhile to ping those teams to let them know.

Reply:
yup will double check with @adhiraj on this